include/vapor/ColorbarPbase.h

```diff
@@ -146,11 +146,13 @@ class PARAMS_API ColorbarPbase : public ParamsBase {
 	static string GetClassType() {
 		return("ColorBarSettingParams");
 	}
-private:
+public:
 
 	static const string _colorbarBackColorTag;
-	static const string _colorbarSizeTag;
-	static const string _colorbarPositionTag;
+	static const string _colorbarSizeXTag;
```

Comment:
Will this break compatibility?

Reply:
In the sense that certain renderers may lose a part of their saved state but the session files will still be 95% ok. It is possible to achieve similar functionality without making these changes. We can discuss pros/cons at our meeting.